wdl/transforms/draft2/src/test/scala/wdl/transforms/wdlwom/WdlSubworkflowWomSpec.scala

```diff
@@ -47,7 +48,7 @@ class WdlSubworkflowWomSpec extends FlatSpec with Matchers {
       """.stripMargin
 
 
-    def innerResolver: Draft2ImportResolver = _ => innerWdl
+    def innerResolver: Draft2ImportResolver = str => Draft2ResolvedImportBundle(innerWdl, str)
```

Comment:
Do these work if you do `def innerResolver: Draft2ImportResolver = Draft2ResolvedImportBundle(innerWdl, _)`?

Reply:
It does. I have updated it 👍